swap_meet/vendor.py

```diff
@@ -1,2 +1,73 @@
 class Vendor:
-    pass
\ No newline at end of file
+
+    def __init__(self, inventory=[]):
+        self.inventory = inventory
+    
+    def add(self, item):
+        if not self.inventory:
+            self.inventory = []
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if not item in self.inventory:
+            return False
+        self.inventory.remove(item)
+        return item
+
+    def get_by_category(self, category):
+        category_list = []
+        for item in self.inventory:
+            if item.category == category:
+                category_list.append(item)
+        return category_list
+
+    def swap_items(self, other_vendor, my_item, other_vendor_item):
+        if not my_item in self.inventory or\
+            not other_vendor_item in other_vendor.inventory:
+            return False
+        my_item_to_swap = self.remove(my_item)
+        other_vendor.add(my_item_to_swap)
+        their_item_to_swap = other_vendor.remove(other_vendor_item)
+        self.add(their_item_to_swap)
```

Comment:
Nice job reusing the instance methods `remove` and `add` that you already wrote!